pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -130,9 +127,16 @@
                             if (LOG.isDebugEnabled()) {
                                 LOG.debug("Successfully acquired zk lock on {}", namespaceBundleZNode);
                             }
+                            final OwnedBundle bundle = new OwnedBundle(
+                                    ServiceUnitZkUtils.suBundleFromPath(namespaceBundleZNode, bundleFactory));
+                            // update the version
+                            localZkCache.getZooKeeper().exists(path, false, (rc1, path1, ctx1, stat) -> {
```

Comment:
I mean when broker-1 is shutting down it [unloads](https://github.com/yahoo/pulsar/blob/master/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java#L342) owned bundles.

>what if broker-1 which owns bundle-1 and due to zk session is lost ephemeral node is deleted and new ephemeral node for bundle-1 will be created by other broker-2.

Isn't in this case: broker-1 still has bundle in its `OwnershipCache` and broker-2 owns same bundle at the same time?